specifications/definition/fieldset/definition.fieldset.schema.json

```diff
@@ -14,8 +14,8 @@
       "type": "string",
       "content": true
     },
-    "summaryLegend": {
-      "title": "Summary Legend",
+    "legendSummary": {
+      "title": "Legend Summary",
```

Comment:
I think the title shouldn't be flipped

`Summary legend`

or if it is...

`Legend (Summary version)`

 ?

Reply:
Was saying it aloud, and thinking that:

`Button tooltip` sounds better than `Tooltip button`?

Reply:
Is it a summary of the legend ('Legend Summary') or a legend for use in a summary (`Legend (Summary version)`)?

Reply:
Fair point! Updated!